directives/nginx-dirs.go

```diff
@@ -0,0 +1,87 @@
+//+build ignore
```

Comment:
Can you add a comment describing what this file does and how to use it? Maybe just explain that it creates txt files with nginx directives?

Reply:
Added README file to explain the rationale and how to use it

https://github.com/caddyserver/nginx-adapter/blob/9872bb2a0ef6a2922233d0bc58be8511e15ea957/directives/README.md